### ISSUE TITLE
VRTK on UWP/Windows Mixed Reality cannot load SDK manager/camera

### ISSUE THREAD

**Issue by lordvard** [ExtendRealityLtd/VRTK#1901]
> **NOTE:** Any issue that does not follow the below template will be immediately closed and not re-opened until the template structure is adhered to.

### Precheck

 * Do not use the issues tracker for help or support (try VRTK Slack channel, Stack Overflow, etc.)
 * For proposing a new feature, please check existing open and closed issues before creating a duplicate
 * For bugs, do a quick search and make sure the bug has not yet been reported
 * Finally, be excellent to each other, and party on dudes!

### Environment

 * Source of VRTK (Unity Asset Store or Github): Github
 * Version of VRTK (Unity Asset Store/Github release number) (Github master commit hash) : 3.3.0 master
 * Version of Unity3D (e.g. Unity 5.4.4f1): 2017.4.11f1
 * Hardware used (e.g. Vive/Oculus): WMR
 * SDK used (e.g. OpenVR/SteamVR/Oculus Utilities): UWP

### Steps to reproduce

Build a scene with VRTK SDK manager set up for WMR in Unity
Build game in VS
Play game
Cry

### Expected behavior

The game should load the VRTK SDK manager and the camera.

### Current behavior

The VRTK SDK manager cannot load and the camera is not activated. Music plays and everything else seems to have loaded fine according to debug logs. Errors thrown in regard to VRTK are:

"Exception thrown at 0x00007FFC6179A388 in Woeful Woebots.exe: Microsoft C++ exception: Il2CppExceptionWrapper at memory location 0x0000007C11EFD720. The script behaviour 'VRTK.VRTK_SDKManager' could not be instantiated!"

"A script behaviour (probably VRTK.VRTK_SDKManager?) has a different serialization layout when loading. (Read 32 bytes but expected 92 bytes)
Did you #ifdef UNITY_EDITOR a section of your serialized properties in any of your scripts?"

**Comment by bddckr** [comment]
Try **not** using the prefab `[VRTK_SDKManager].prefab` - instead set it up yourself. Does that help?

If not do me a favor and try changing this to `private`:
https://github.com/thestonefox/VRTK/blob/527e6225f7d9611403d557b82763267655354deb/Assets/VRTK/Source/Scripts/Utilities/SDK/VRTK_SDKManager.cs#L346
Then additionally comment out all of this:
https://github.com/thestonefox/VRTK/blob/d88c9eafc31975bb9566e5747c12e48e5f75fa3d/Assets/VRTK/Source/Editor/VRTK_SDKManagerEditor.cs#L407-L416

Let me know if any of this helps or not, thanks!

**Comment by lordvard** [comment]
Thank you very much, Christopher! I will give this a try right away and let you know how it goes.

**Comment by lordvard** [comment]
Okay, so I first tried to just change the scripts as you mentioned, and that did not help, still in a dark room with just the music playing. The error in the debug window is not there, but that's been the case a few times earlier too, hasn't been consistently showing up oddly enough. After trying with just the scripts updated, I got rid of the prefab SDK manager and set up a new one from scratch. Unfortunately that also did not help. 

I tried this before and after these suggestions, but by setting the WindowsMR object to be active in the SDK manager hierarchy I can get the camera to work, but then the controllers don't work. I tried this in the editor and they work there, so clearly something isn't being activated in terms of VRTK's SDK when a solution is built in VS.
